packages/backend-app-api/api-report.md

```diff
@@ -4,14 +4,14 @@
 
 ```ts
 import { BackendFeature } from '@backstage/backend-plugin-api';
-import { CacheService } from '@backstage/backend-plugin-api';
 import { ConfigService } from '@backstage/backend-plugin-api';
-import { DatabaseService } from '@backstage/backend-plugin-api';
 import { ExtensionPoint } from '@backstage/backend-plugin-api';
 import { HttpRouterService } from '@backstage/backend-plugin-api';
 import { LifecycleService } from '@backstage/backend-plugin-api';
 import { LoggerService } from '@backstage/backend-plugin-api';
 import { PermissionsService } from '@backstage/backend-plugin-api';
+import { PluginCacheManager } from '@backstage/backend-common';
```

Comment:
Does it still have to import these from `@backstage/backend-common` or can we not just import from the `backend-plugin-api`?

Reply:
Or maybe you wanted to leave that for the other ticket?